packages/gatsby-plugin-sharp/src/process-file.js

```diff
@@ -75,6 +76,31 @@ const argsWhitelist = [
 exports.processFile = (file, transforms, options = {}) => {
   let pipeline
   try {
+    // adds gatsby cloud image service to gatsby-sharp
+    // this is an experimental api so it can be removed without any warnings
+    if (process.env.GATSBY_CLOUD_IMAGE_SERVICE_URL) {
+      let cloudPromise
+
+      return transforms.map(transform => {
+        if (!cloudPromise) {
+          cloudPromise = got
+            .post(process.env.GATSBY_CLOUD_IMAGE_SERVICE_URL, {
+              body: {
+                file,
+                transforms,
```

Comment:
you are sure to send all `transforms` instead of `transform`?

Reply:
yeah we do 😄 Thanks for asking.

Reply:
🤦🏻‍♂️ Ok. Sure. I see now It is only called once for a transactions